### PR TITLE
[FluidBio] Support tables and flow rate in parabolic inlet

### PR DESCRIPTION
**Pull request by rubenzorrilla**
**📝 Description**
This PR gives support for CSV tables and flow rates in the parabolic inlet process.

### CODE REVIEW

File: applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_parabolic_inlet_process.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Importing the Kratos Library
+from doctest import ELLIPSIS_MARKER
+from logging import Logger
 import KratosMultiphysics
+from KratosMultiphysics.read_csv_table_utility import ReadCsvTableUtility
```
Comment (miguelmaso) [inline applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_parabolic_inlet_process.py:4]: nice!

Comment (rubenzorrilla) [reply applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_parabolic_inlet_process.py:4]: :wink:

File: applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_parabolic_inlet_process.py [hunk 2]
```diff
@@ -89,18 +101,38 @@ def ExecuteInitialize(self):
...
-                KratosBio.ParabolicProfileUtilities.ImposeParabolicInlet(inlet_model_part, self.max_value)
+                KratosBio.ParabolicProfileUtilities.ImposeParabolicInlet(inlet_model_part, self.max_value, max_value_factor)
+            elif self.max_value_is_function:
+                KratosBio.ParabolicProfileUtilities.ImposeParabolicInlet(inlet_model_part, self.max_value_function, max_value_factor)
+            elif self.max_value_is_table:
+                current_max_value = self.table.GetValue(current_time)
+                KratosBio.ParabolicProfileUtilities.ImposeParabolicInlet(inlet_model_part, current_max_value, max_value_factor)
             else:
-                KratosBio.ParabolicProfileUtilities.ImposeParabolicInlet(inlet_model_part, self.max_value_function)
+                raise Exception("Wrong maximum value data type.")
```
Comment (EduardGomezEscandell) [inline applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_parabolic_inlet_process.py]: TypeError may be more precise

**Review by EduardGomezEscandell — DISMISSED** [review]
Minor thing, non-blocking

File: applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_parabolic_inlet_process.py (hunk 2 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_parabolic_inlet_process.py]: Done!

**Review by EduardGomezEscandell — APPROVED** [review]

**Pull request merged**